Sweeper/Models/Pin.swift

```diff
@@ -79,3 +60,30 @@ class Pin: PFObject, PFSubclassing {
         }
     }
 }
+
+class PinMarker: GMSMarker {
```

Comment:
Can we move this to it's own class file?

Reply:
Yea, I'll do that early next week.